CHANGELOG.md

```diff
@@ -1,3 +1,82 @@
+## Release 0.11.0
+
+New features:
+
+- UI Terminal to attach to Docker containers and see their output or `EXEC` a shell on them
+	[#650](https://github.com/weaveworks/scope/pull/650)
+  [#735](https://github.com/weaveworks/scope/pull/735)
+  [#726](https://github.com/weaveworks/scope/pull/726)
+- Added `scope version` command
+	[#750](https://github.com/weaveworks/scope/pull/750)
+- Various CPU usage gains for probe
+	[#742](https://github.com/weaveworks/scope/pull/742)
+	[#741](https://github.com/weaveworks/scope/pull/741)
+	[#737](https://github.com/weaveworks/scope/pull/737)
+- Show hostname of app you are connected to
+	[#707](https://github.com/weaveworks/scope/pull/707)
+- Add host memory and CPU usage metrics
+	[#711](https://github.com/weaveworks/scope/pull/711)
+- Add json support to app POST /api/report
+	[#722](https://github.com/weaveworks/scope/pull/722)
+- Add basic security warning.
+	[#703](https://github.com/weaveworks/scope/pull/703)
+- Update the docker version we embed into the scope image to 1.6.2 in sync with weave 1.3 changes.
+	[#702](https://github.com/weaveworks/scope/pull/702)
+- Show a spinner while node details are loading
+	[#691](https://github.com/weaveworks/scope/pull/691)
+- Procedural colors for nodes
```

Comment:
This probably need a better description - "Make node colorrs deterministic based on node name."